tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -534,6 +534,13 @@ def from_tensors(tensors):
     >>> list(dataset.as_numpy_iterator())
     [(array([1, 2, 3], dtype=int32), b'A')]
 
+    >>> # `from_tensors` does not change the shape of input tensor.
+    >>> # use `from_tensor_slices` to slice the input tensor.
```

Comment:
Can we put the comparison to `from_tensor_slices` in a sentence right after the first line? It's out of place here since the example code doesn't use `from_tensor_slices`. e.g.

```
"""Creates a `Dataset` with a single element, comprising the given tensors.

`from_tensors` produces a dataset containing only a single element. To slice the input 
tensor into multiple elements, use `from_tensor_slices` instead.
```

Instead we can describe the example according to what it does, e.g. "You can use `from_tensors` to produce a dataset which repeats the same example many times".

Reply:
Yes changed it. Also changed `from_tensor_slices` example so that readers will be able to differentiate both methods in repeat() examples.